README.md

```diff
@@ -24,8 +24,129 @@ Please open a github issue.
 
 ## Getting Started
 
-....
+### How to Integrate
 
-## What are the tests?
 
-....
\ No newline at end of file
+Create the following file:
+
+`$PROJECT_DIR/tests/static/test_gardener.py`
+
+```python
+import os
+from py_gardener.StaticTestBase import StaticTestBase
+
+
+class TestGardener(StaticTestBase):
+    ROOT_DIR = os.path.join(os.path.dirname(__file__), '..', '..')
+    TEST_DIR = os.path.join(ROOT_DIR, "tests")
+    LIB_DIR = os.path.join(ROOT_DIR, "service_acl_data")
+
+```
```

Comment:
I would just reference the file here instead of copying something that will be outdated.